consensus-types/types/block.go

```diff
@@ -153,16 +148,21 @@ func (b *BeaconBlock) GetStateRoot() common.Root {
 	return b.StateRoot
 }
 
-// Version identifies the version of the BeaconBlock.
-func (b *BeaconBlock) Version() common.Version {
-	return b.BbVersion
-}
-
 // SetStateRoot sets the state root of the BeaconBlock.
 func (b *BeaconBlock) SetStateRoot(root common.Root) {
 	b.StateRoot = root
 }
 
+// GetForkVersion identifies the version of the BeaconBlock.
+func (b *BeaconBlock) GetForkVersion() common.Version {
+	return b.forkVersion
+}
+
+// SetForkVersion sets the fork version of the BeaconBlock
+func (b *BeaconBlock) SetForkVersion(version common.Version) {
```

Comment:
not a fan of this as I see its only required in tests. If thats the case we can keep but would prefer noting that its only used for testing